_episodes/02-working-with-environments.md

```diff
@@ -433,7 +428,7 @@ longer need an existing environment and just want to get rid of cruft on your ma
 the reason, the command to delete an environment is the following.
 
 ~~~
-$ conda remove --name my-first-conda-env --all
+$ conda remove --name first-conda-env --all
```

Comment:
Have we created an environment called this, by this point i.e. will this code run?

Reply:
Good catch, we hadn't. That is what I get for trying to be clever. I've opted to remove `python3-env` which is definitely created but not used anywhere else